utils/apiValidators.ts

```diff
@@ -0,0 +1,43 @@
+import { ApiError } from './types'
+import {
+  validateAttribute,
+  validateCategory,
+  validateItemDefinition,
+  validateObjectId,
+  validateUser,
+  ValidationResult,
+} from './validators'
+
+const BAD_REQUEST_BODY_PREFIX = 'Bad Request Body: '
+
+export function apiObjectIdValidation(id: any) {
```

Comment:
why are we accepting anything here if the `validateObjectId` function is expecting a string? I think this type should be `string`

Reply:
fixed